app/_features/event/components/event-form.tsx

```diff
@@ -0,0 +1,493 @@
+                    <p className="mt-1 text-xs">560 x 280, Aspect ratio 2:1</p>
+                  </div>
+                )}
+              </div>
+              <div className="flex w-full flex-wrap gap-2 sm:flex-nowrap">
+                <div className="w-full sm:basis-1/3">
+                  <FormWithIcon
+                    title="Event Date"
+                    icon={<CalendarIcon width={24} height={24}></CalendarIcon>}
+                    isReadOnly={true}
+                    onClickForm={() => {
+                      document.dispatchEvent(
+                        new CustomEvent('open:event-date-picker-modal')
+                      );
+                    }}
+                    value={new Intl.DateTimeFormat('en-GB').format(date)}
+                    onClickIcon={() => {
+                      document.dispatchEvent(
+                        new CustomEvent('open:event-date-picker-modal')
+                      );
+                    }}
+                  ></FormWithIcon>
+                </div>
+                <div className="w-full sm:basis-1/3">
+                  <FormWithIcon
+                    value={`${startTime.hour}:${startTime.minute}`}
+                    title="Start Time"
+                    icon={
+                      <ClockFillIcon width={24} height={24}></ClockFillIcon>
+                    }
+                    isReadOnly={true}
+                    onClickForm={() => {
+                      document.dispatchEvent(
+                        new CustomEvent('open:event-time-picker-start-modal')
+                      );
+                    }}
+                    onClickIcon={() => {
+                      document.dispatchEvent(
+                        new CustomEvent('open:event-time-picker-start-modal')
+                      );
+                    }}
+                  ></FormWithIcon>
+                </div>
+                <div className="w-full sm:basis-1/3">
+                  <FormWithIcon
+                    title="End Time"
+                    value={`${endTime.hour}:${endTime.minute}`}
+                    icon={
+                      <ClockFillIcon width={24} height={24}></ClockFillIcon>
+                    }
+                    isReadOnly={true}
+                    onClickForm={() => {
+                      document.dispatchEvent(new CustomEvent(setEndTimeEvent));
+                    }}
+                    onClickIcon={() => {
+                      document.dispatchEvent(new CustomEvent(setEndTimeEvent));
+                    }}
+                  ></FormWithIcon>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  );
+}
+
+function ComingSoonModal({ event }: { event: string }) {
+  const { isOpen, onOpen, onClose, onOpenChange } = useDisclosure();
+
+  const openModal = useCallback(() => {
+    console.log('open coming soon modal');
```

Comment:
Remove this logging

Reply:
removed the unnecessary console log https://github.com/inlivedev/inlive-room/pull/130/commits/06de4972dc0b8b36258652c019bd4a9acd9245ff